releasenotes/notes/remove-pulse-calibrations-4486dc101b76ec51.yaml

```
@@ -0,0 +1,37 @@
+---
+upgrade_circuits:
+  - |
+    As part of Pulse removal in Qiskit 2.0, the ``calibrations`` property has been removed
+    from the :class:`.QuantumCircuit`, :class:`.DAGCircuit` and :class:`.DAGDependency` classes.
+    In addition, the method ``has_calibration_for`` has been removed from the :class:`.QuantumCircuit` and
+    :class:`.DAGCircuit` classes and ``add_calibration`` has been removed from :class:`.QuantumCircuit`.
+upgrade_transpiler:
+  - |
+    As part of Pulse removal in Qiskit 2.0, all pulse and calibration related functionality
+    in the transpiler has been removed. This includes the following:
+
+    Passes that have been removed:
+
+      * ``qiskit.transpiler.passes.PulseGates``
+      * ``qiskit.transpiler.passes.ValidatePulseGates``
+      * ``qiskit.transpiler.passes.RXCalibrationBuilder``
+      * ``qiskit.transpiler.passes.RZXCalibrationBuilder``
+      * ``qiskit.transpiler.passes.RZXCalibrationBuilderNoEcho``
+      * ``qiskit.transpiler.passes.EchoRZXWeylDecomposition``
+
+    The ``inst_map`` argument has been removed from the following elements:
+
+      * The :func:`.generate_preset_pass_manager` and :func:`.transpile` functions
+      * The :meth:`.Target.from_configuration` method
+      * The constructor of the :class:`.PassManagerConfig` class
+
+    Calibration support has been removed:
+
+      * ``calibration`` has been removed from the :class:`.InstructionProperties` 's constructor and is no longer a property of that class.
+      * The ``has_calibration``, ``get_calibration``, ``instruction_schedule_map`` and ``update_from_instruction_schedule_map`` methods have been removed from the :class:`.Target` class.
+
+
+upgrade_misc:
+  - |
+    As part of Pulse removal in Qiskit 2.0, the ``sequence`` and ``schedule_circuit`` functions
```

I think it's better to specify the module here to avoid confusions with other ``sequence`` and ``schedule`` (they are all being removed anyways):

```suggestion
    As part of Pulse removal in Qiskit 2.0, the ``sequence`` and ``schedule_circuit`` functions from :mod:`.qiskit.scheduler`
```

---

You're right.